images/common/utils.sh

```
@@ -236,18 +236,25 @@ function openvpn_config_checksum {
 
 function openvpn_config_download {
-	curl --silent --retry 10 --retry-delay 5 --retry-max-time 300\
-		--insecure --output vpn.tar.gz \
-		${API_INTERNAL}/controller/vpn/download-config/$UUID/?key=$KEY
+	curl --silent --retry 10 --retry-delay 5 --retry-max-time 300 --insecure --output vpn.tar.gz \
+		"${API_INTERNAL}/controller/vpn/download-config/$UUID/?key=$KEY"
 	curl --silent --insecure --output checksum \
-		${API_INTERNAL}/controller/vpn/checksum/$UUID/?key=$KEY
+		"${API_INTERNAL}/controller/vpn/checksum/$UUID/?key=$KEY"
 	tar xzf vpn.tar.gz
 	chmod 600 *.pem
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)